build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -1158,7 +1167,11 @@ CONFIG_LIBC_STRERROR=y
 CONFIG_LIBC_TMPDIR="/tmp"
 CONFIG_LIBC_MAX_TMPFILE=32
 CONFIG_ARCH_LOWPUTC=y
-# CONFIG_LIBC_LOCALTIME is not set
+CONFIG_LIBC_LOCALTIME=y
```

Comment:
As far as I know, we disable this.
@ewoodev Could you confirm this?

Reply:
This need to be enabled.

Reply:
@ewoodev this should be enabled in all defconfigs then?

Reply:
yes

```
CONFIG_LIBC_LOCALTIME=y
CONFIG_LIBC_TZ_MAX_TIMES=320
CONFIG_LIBC_TZ_MAX_TYPES=20
CONFIG_LIBC_TZDIR="/etc/zoneinfo"
# CONFIG_LIBC_DOWNLOAD_ZONEINFO is not set
```

Reply:
Updated all